src/templateSources.js

```diff
@@ -119,6 +119,6 @@ anonymousTemplate.prototype.text = function(/* valueToWrite */) {
     }
 };
 
-ko.exportSymbol('templateSources', ko.templateSources);
-ko.exportSymbol('templateSources.domElement', ko.templateSources.domElement);
-ko.exportSymbol('templateSources.anonymousTemplate', ko.templateSources.anonymousTemplate);
+//ko.exportSymbol('templateSources', ko.templateSources);
```

Comment:
We can just delete these exportSymbols - it's trivial to grep the original source for exports.